lib/features/auto-place/BpmnAutoPlaceUtil.js

```diff
@@ -135,4 +135,26 @@ export function getDataElementPosition(source, element) {
   };
 
   return findFreePosition(source, element, position, generateGetNextPosition(nextPositionDirection));
+}
+
+/**
+ * Always have the new element in frame
+ */
+export function getScrollOffset(element, boundingBox) {
+  var elementTrbl = asTRBL(element);
+  var bbTrbl = asTRBL(boundingBox);
+
+  // Bigger padding to accommodate the context pad
+  var paddingRight = 80;
```

Comment:
where do we get this value from?

Reply:
It's the css width of the context-pad (72px) + additional padding

Reply:
Ok got it, thanks for the clarification 👍

Reply:
I also discussed it with @philippfromme and we decided to make the padding consistent (80 to all sides) to cover most use-cases